mmcv/ops/modulated_deform_conv.py

```diff
@@ -352,3 +355,69 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
         super()._load_from_state_dict(state_dict, prefix, local_metadata,
                                       strict, missing_keys, unexpected_keys,
                                       error_msgs)
+
+
+@CONV_LAYERS.register_module(
+    'DCNv2' if IS_MLU_AVAILABLE else 'DCNv2_disabled',
+    force=True if IS_MLU_AVAILABLE else False)
+class ModulatedDeformConv2dPack_MLU(nn.modules.Module):
```

Comment:
```suggestion
if IS_MLU_AVAILABLE:
    @CONV_LAYERS.register_module('DCNv2', force=True)
    class ModulatedDeformConv2dPack_MLU(nn.modules.Module):
```

Reply:
Please add a description to explain why we need to implement a new class for MLU.

Reply:
> 

This modification will cause an error to be reported when running on a non-MLU device, because I have imported it in mmcv/ops/__init__.py